docs/manage/ml/_index.md

```diff
@@ -5,18 +5,25 @@ weight: 40
 type: "docs"
 tags: ["data management", "data", "services"]
 no_list: true
-description: "Use Viam's built-in machine learning capabilities to train image classification models and deploy these models to your robots"
+description: "Use Viam's built-in machine learning capabilities to train image classification models and deploy these models to your robots."
 # SME: Aaron Casas
 ---
 
-Machine learning is an approach to programming where you provide an algorithm with a large data set, and train it to discover patterns in that data on its own.
-The more data you provide to it, the better it becomes at identifying patterns across that data set, and the more accurate it becomes at guessing characteristics of any new data provided.
+Viam includes a built-in [machine learning (ML) service](/services/ml/) which provides your robot with the ability to learn from data and adjust its behavior based on insights gathered from that data.
+Common use cases include object detection, image classification, natural language processing, and speech recognition and synthesis, but your robot can make use of machine learning with nearly any kind of data.
 
-For example, say you have a collection of pictures of your cat.
-As you train your algorithm on just a few pictures, it might not be able to determine that a picture of your cat's face and a picture of your cat sleeping are the same cat.
-As you train the algorithm on more and more pictures, it might be able to recognize your cat from any angle, or to differentiate between multiple different cats.
+Viam natively supports [TensorFlow Lite](https://www.tensorflow.org/lite) ML models, but you can use other models as well as long as your models adhere to the [model requirements](/services/ml/#tflite_cpu-limitations).
+
+To make use of ML models with your robot, you can use the built-in ML model service to train image classification models for object detection and classification or create a [modular resource](/program/extend/modular-resources/) to integrate it with your robot.
+
+### Object detection and classification
+
+Once you have [created the ML model service](/services/ml/#create-an-ml-model-service) for your robot, you can [train image classification models](train-model/) to enable it to detect people, animals, plants or other objects with bounding boxes and perform actions when they are detected.
+
+![Gif of a dog being labeled](/tutorials/img/pet-treat-dispenser/app-data-images.png)
+
+When training machine learning models, it is important to supply a variety of different data about the subject. In the case of object detection, it is important to provide images of the object in different situations, such as from different angles or in different lighting situations. The more varied the provided data set, the more accurate the resulting model becomes.
```

Comment:
This should probably now actually move to the train page - it no longer feels like it fits here

Reply:
Moved!